google/cloud/spanner_v1/instance.py

```diff
@@ -190,6 +211,44 @@ def name(self):
         """
         return self._client.project_name + "/instances/" + self.instance_id
 
+    @property
+    def processing_units(self):
+        """Processing units used in requests.
+
+        :rtype: int
+        :returns: The number of processing units allocated to this instance.
+        """
+        return self._processing_units
+
+    @processing_units.setter
+    def processing_units(self, value):
+        """Sets the processing units for requests. Affects node_count.
+
+        :param value: The number of processing units allocated to this instance.
+        """
+        self._processing_units = value
+        self._node_count = value // PROCESSING_UNITS_PER_NODE
+
+    @property
+    def node_count(self):
+        """Node count used in requests.
+
+        :rtype: int
+        :returns node_count:
```

Comment:
nit: returns don't need a name
```suggestion
        :returns:
```